Source/Diagnostics/ComputeDiagFunctors/BackTransformParticleFunctor.cpp

```diff
@@ -149,7 +149,6 @@ BackTransformParticleFunctor::operator () (PinnedMemoryParticleContainer& pc_dst
             }
         }
     }
-    totalParticleCounter = pc_dst.TotalNumberOfParticles();
```

Comment:
As discussed with @RevathiJambunathan, this is called for every BTD snapshot and at every PIC iteration, and generates MPI calls. It is therefore an expensive operation.